content/strategy-goals/strategy/enablement/frontend-platform/index.md

```diff
@@ -13,37 +13,14 @@ Using and developing Sourcegraph is effortless.
 - We have a frictionless developer experience that empowers our frontend engineers to achieve the maximum value from the time spent on development.
 - We have an intuitive, performant, and consistent user experience that makes Sourcegraph an indispensable part of every developer workflow.
 
-## Goals for FY22
+## Roadmap
```

Comment:
Thanks, I wonder if there's a better source of truth for this that doesn't require us manually updating this text every quarter.

Maybe we could just link to the Enablement roadmap? https://github.com/orgs/sourcegraph/projects/214/views/14 WDYT?